CHANGELOG.md

```diff
@@ -26,6 +42,10 @@
 - Added a `MockConnectionHandler` implementation for testing code which requires the world to be populated with SpatialOS entities.
 - Added tests for `StandardSubscriptionManagers` and `AggregateSubscription`.
 - Re-added tests for Reader/Writer injection criteria and Monobehaviour enabling.
+- Reactive components have been isolated and can be enabled or disabled.
+- Subscriptions API has been added, this allows you to subscribe to Reader, Writers, et al outside of Monobehaviours
```

Comment:
Can subscribe to anything for which a manager / publisher has been defined. This is what is backing Require.